src/App.css

```diff
@@ -1,39 +1,50 @@
-.App {
-  text-align: center;
+.calculator {
+  width: 25%;
+  margin: auto;
+  padding: 10% 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.screen {
+  background-color: #858694;
+  text-align: right;
+  margin-bottom: 0;
+  height: 3rem;
+  color: white;
+  font-weight: bold;
 }
```

Comment:
- [ ] The text in the screen element is not positioned properly. Kindly compare it with the original design and make changes to your styling. :pray: 

![image](https://user-images.githubusercontent.com/64894927/133237061-e2df3bf8-2bad-4f56-831e-e906b44fed44.png)
![image](https://user-images.githubusercontent.com/64894927/133237103-f548839c-0f84-4fa5-8762-ce95780dd67a.png)

Reply:
This issue is resolved. Job well done! :clap: :+1: